examples/native/components/ControlExamples/Color/Color.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FunctionComponent } from 'react';
+import { View, StyleSheet, Text, ColorValue } from 'react-native';
+
+interface ButtonProps {
+  color: ColorValue;
+}
+
+const styles = StyleSheet.create({
+  button: { paddingHorizontal: 16, paddingVertical: 8 },
+});
+
+export const Color: FunctionComponent<ButtonProps> = ({ color }) => (
```

Comment:
Trivial, but we might want to not use `FunctionComponent` or `FC`, as the React core team seems to lean on the side of "don't use it": https://github.com/facebook/create-react-app/pull/8177

Reply:
I'm fine with either way. Happy to remove it 👍.

Reply:
I guess we should remove FC / FunctionComponent type annotations across the rest of the codebase just to be consistent. I'm fine with just including the prop types and leaving the function type implicit so like:

`const MyComponent = ({X}:Props) => <View/>`

Reply:
Sounds great! We could tackle this in a separate PR, e.g. by running https://github.com/gndelia/codemod-replace-react-fc-typescript on the codebase.

Reply:
We will open an issue to run this codemod